clouddriver-ecs/src/integration/java/com/netflix/spinnaker/clouddriver/ecs/test/EcsControllersSpec.java

```diff
@@ -30,8 +30,8 @@
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.Map;
-import org.junit.Test;
 import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
```

Comment:
Is this intentional?

Reply:
Yes! ToC [guidance](https://github.com/spinnaker/clouddriver/pull/4827#issuecomment-678526751) is to use Junit5 for these tests. See [K8s integ tests](https://github.com/spinnaker/clouddriver/blob/master/clouddriver-kubernetes/src/integration/java/com/netflix/spinnaker/clouddriver/kubernetes/it/DeployManifestIT.java#L33-L34) for and existing example.